packages/nextjs-mf/src/plugins/container/InvertedContainerRuntimeModule.ts

```diff
@@ -31,6 +31,9 @@ class InvertedContainerRuntimeModule extends RuntimeModule {
       return '';
     }
 
+    if (this.chunk.runtime === 'webpack-api-runtime') {
```

Comment:
When will the chunk.name equals 'webpack-api-runtime' ?

Reply:
When runtimeChunk is function or when entry point defines "runtime" value for entry. 

This is workaround because embedded container crashes the api layer due to many runtime chunks generated from a single build. 

In future I think I will change to single runtime chunk runtime plugin to solve the category of problems. But for now they will prevent 500 errors

Reply:
gotcha, LGTM!